sqlengine/mysql_engine.go

```diff
@@ -74,18 +75,19 @@ func (d *MySQLEngine) CreateUser(bindingID, dbname string) (username, password s
 		"TRIGGER",
 	}
 
-	createUserStatement := "CREATE USER '" + username + "'@'%' IDENTIFIED BY '" + password + "'"
+	createUserStatement := "CREATE USER '" + username + "'@'%' IDENTIFIED BY '" + password + "';"
 	d.logger.Debug("create-user", lager.Data{"statement": createUserStatement})
```

Comment:
This isn't in scope for this PR, but this is currently logging the password, which we don't want. We changed this for Postgres in #8. I'll raise a follow-up story to fix this.